Yosemite/Yosemite/Stores/InAppPurchaseStore.swift

```diff
@@ -5,8 +5,11 @@ import Networking
 
 public class InAppPurchaseStore: Store {
     private var listenTask: Task<Void, Error>?
+    private let remote: InAppPurchasesRemote
+    private var useBackend = true
```

Comment:
I am wondering, what will be the usage of this property? How are we going to set it? I see that it is a private var, but I don't see any place where we change its value :)

Reply:
Initially I wanted to make it configurable through the debug screen, but it got a bit complicated. I left that there so it was easier to change behavior during development/testing.